run/runner.go

```diff
@@ -79,7 +80,7 @@ func (r *Runner) fullRun(id int) (*Result, error) {
 		return nil, err
 	}
 	log.Printf("RUN %v: Starting full run with hash %v", id, hash)
-	result, err := r.run(id, rawList, hash)
+	result, err := r.run(id, true, rawList, hash)
```

Comment:
For example while reading this code, I would have to jump back to the definition to decipher what the "true" means